sympy/stats/crv_types.py

```diff
@@ -2195,6 +2196,104 @@ def Normal(name, mean, std):
             MultivariateNormalDistribution, name, mean, std)
     return rv(name, NormalDistribution, (mean, std))
 
+
+#-------------------------------------------------------------------------------
+# Inverse Normal distribution ----------------------------------------------------------
+
+
+class NormalInverseDistribution(SingleContinuousDistribution):
+    _argnames = ('mean', 'shape')
+
+    @property
+    def set(self):
+        return Interval(0, oo)
+
+    @staticmethod
+    def check(mean, shape):
+        _value_check(shape > 0, "Shape parameter must be positive")
+        _value_check(mean > 0, "Mean must be positive")
+
+    def pdf(self, x):
+        return exp(-self.shape*(x - self.mean)**2 / (2*x*self.mean**2)) * sqrt(self.shape/((2*pi*x**3)))
+
+    def sample(self):
+        scipy = import_module('scipy')
+        if scipy:
+            from scipy.stats import invgauss
+            return invgauss.rvs(float(self.mean/self.shape), 0, float(self.shape))
+        else:
+            raise NotImplementedError('Sampling the Inverse Normal Distribution requires Scipy.')
+
+    def _standardNormal_cdf(self, x):
+        return erf(sqrt(2)*(x - S.One)/2)/2 + S.Half
+
+    def _cdf(self, x):
+        mean, shape = self.mean, self.shape
+
+        first_term = self._standardNormal_cdf(sqrt(shape/x) * ((x/mean - S.One)))
+        second_term = exp(2*shape/mean) * self._standardNormal_cdf(-sqrt(shape/x)*(x/mean + S.One))
+
+        return  first_term + second_term
+
+    def _characteristic_function(self, t):
+        mean, shape = self.mean, self.shape
+        return exp((shape/mean)*(1 - sqrt(1 - (2*mean**2*I*t)/shape)))
+
+    def _moment_generating_function(self, t):
+        mean, shape = self.mean, self.shape
+        return exp((shape/mean)*(1 - sqrt(1 - (2*mean**2*t)/shape)))
+
+
+def NormalInverse(name, mean, shape):
+    r"""
+    Create a continuous random variable with an Inverse Normal distribution.
+
+    The density of the Inverse Normal distribution is given by
+
+    .. math::
+        f(x) := \sqrt{\frac{\lambda}{2\pi x^3}} e^{-\frac{\lambda(x-\mu)^2}{2x\mu^2}}
+
+    Parameters
+    ==========
+
+    mu : Positive number representing the mean
+    lambda : Positive number representing the shape parameter
+
+    Returns
+    =======
+
+    A RandomSymbol.
+
+    Examples
+    ========
+
+    >>> from sympy.stats import NormalInverse, density, E, std
+    >>> from sympy import Symbol
+
+    >>> mu = Symbol("mu", positive=True)
+    >>> l = Symbol("lambda", positive=True)
+    >>> z = Symbol("z", positive=True)
+    >>> X = NormalInverse("x", mu, l)
+
+    >>> density(X)(z)
+    sqrt(2)*sqrt(lambda)*exp(-lambda*(-mu + z)**2/(2*mu**2*z))/(2*sqrt(pi)*z**(3/2))
+
+    >>> E(X)
+    mu
+
+    >>> std(X)
```

Comment:
You can include `std(X).expand()` and `skewness(X).expand()` as suggested in https://github.com/sympy/sympy/pull/16814#discussion_r283256233.

Reply:
Why is `skewness(X)` no longer part of the docstring examples?

Reply:
Added. I accidentally removed it. Thanks!